src/helpers/translations/translations.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  locale: "en",
+  crypto: {
+    ethereum: "etherereum",
+    tokens: "tokens",
+    nft: "Non-Fungible Token",
+    whatIsANFT:
+      " is a way to ensure digital assets are unique and easily tradable on a blockchain.",
```

Comment:
Think this is actually a better description for muggle, as it goes into no details and simply says what it is

Reply:
Theyre pretty much the same just reworded for the different heading, It's a bit clearer when run. Its more a grammer thing